### PR TITLE
Approval flow

### PR DESCRIPTION
**Pull request by ryanlabouve**
Ref: 
* https://app.asana.com/0/home/1201981678382625/1202233470842632
* https://app.asana.com/0/home/1201981678382625/1202233470842624
* https://app.asana.com/0/home/1201981678382625/1202233754163346

### CODE REVIEW

File: app/components/borrow-flow/repay.tsx [hunk 1]
```diff
@@ -187,7 +187,11 @@ export default function Repay({
                       try {
                         setIsEnabling(true);
                         // @ts-ignore existence of signer is gated above.
-                        await enable(signer, row.token, row.cToken);
+                        await enable(
```
Comment (ryanlabouve) [inline app/components/borrow-flow/repay.tsx:190]: This is the good part 1/2

File: app/components/deposit-flow/deposit.tsx [hunk 2]
```diff
@@ -182,7 +182,11 @@ export default function Deposit({
                     try {
                       setIsEnabling(true);
                       // @ts-ignore existence of signer is gated above.
-                      await enable(signer, row.token, row.cToken);
+                      await enable(
```
Comment (ryanlabouve) [inline app/components/deposit-flow/deposit.tsx:185]: This is the good part 2/2

File: app/components/account-summary.tsx [hunk 3]
```diff
@@ -66,14 +66,22 @@ export default function AccountSummary() {
...
+              )}
+              {!netApy && <div className="text-5xl font-light">--</div>}
               <div className="absolute top-0 right-0">
-                <Ring
-                  percent={
-                    totalSupplyBalanceInUsd /
-                    (borrowBalance + totalSupplyBalanceInUsd)
-                  }
-                />
+                {}
```
Comment (johnmosesman) [inline app/components/account-summary.tsx:74]: Is this `{}` supposed to be here?

File: app/components/deposit-flow/deposit.tsx (hunk 2 above)
Comment (johnmosesman) [reply app/components/deposit-flow/deposit.tsx:185]: `// @ts-ignore existence of signer is gated above.`

well if it isn't past us to bite us again

File: app/components/two-panes/market-borrow-row.tsx [hunk 4]
```diff
@@ -3,8 +3,7 @@ import ReactModal from "react-modal";
 import type { Market } from "~/types/global";
 import BorrowFlow from "../borrow-flow";
 
-// TODO: Do I need to set this?
-// ReactModal.setAppElement("#m");
+ReactModal.setAppElement("#m");
```
Comment (johnmosesman) [inline app/components/two-panes/market-borrow-row.tsx:6]: lulz was something broken here or what does this do?

**Review by johnmosesman — APPROVED** [review]
couple questions but gonna merge to get these fixes in

**Pull request merged**